C24_WMDE_Mobile_DE_14

moiikana opened the pull request:
- based on ctrl 13
- var shows UOF link in minibanner

- [x] tests need to be fixed

https://phabricator.wikimedia.org/T382481

KaiNissen commented on campaign_info.toml:
```diff
@@ -4,20 +4,20 @@
 # A Campaign with global settings
 [desktop]
-campaign_tracking = "03-ba-210802"
+campaign_tracking = "04-ba-210812"
```
The start date was moved to Aug 20th. Production banners were already changed, but should be reproduced here.

moiikana committed:
Prepare campaign for C24_WMDE_Mobile_DE_14

- based on ctrl 13

https://phabricator.wikimedia.org/T382481

moiikana committed:
Prepare CTRL for C24_WMDE_Mobile_DE_14

- based on ctrl 13

moiikana committed:
Prepare VAR for C24_WMDE_Mobile_DE_14

- copy minibanner and minibanner styles from 14 ctrl

moiikana committed:
Implement VAR for C24_WMDE_Mobile_DE_14

- add button for UOF to minibanner

- use new 2024 UOF component

- add tests

https://phabricator.wikimedia.org/T382481

moiikana commented:
CI for this change is passing,
but https://github.com/wmde/fundraising-banners/pull/676 needs to be merged
